lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    httpsmissing: 'Solo se permiten orígenes seguros (https) para compartir la pantalla.',
+    ieBrowserIssue: 'Internet Explorer no es compatible, Congrea es totalmente compatible con Chrome y Firefox.',
+    ios7support: 'Para Apple, Virtual Class es compatible con iOS 8 y versiones superiores.',
+    iosAudEnable: 'Toque aquí para habilitar el audio.',
+    InternalError: 'Asegúrese de que no se esté utilizando la misma cámara web <br /> en varios navegadores o varias aplicaciones. ',
+    invalidcmid: 'ID del módulo del curso no válido',
+    invalidurl: 'URL inválida',
+    indvprogress: 'Tarea actual',
+    InsertimageURL: 'Insertar URL de la imagen',
+    JoinClassMsg: 'Únete al aula',
+    JoinSession: 'Únete a la session',
+    keymissing: 'Su software(LMS/CMS)\'s falta la Key',
+    Line: 'Línea',
+    lowBandWidthSpeed: 'Su ancho de banda es demasiado bajo, estamos deteniendo la reproducción de video. Aún podrá escuchar el audio y ver la pantalla.',
+    maxCommonChat: 'Mostrar ventana de chat',
+    Max_rooms: 'Error: Límite máximo de salas alcanzado',
+    Max_users: 'Error: Límite máximo de usuarios alcanzado',
+    media: 'Media',
+    mediumBandWidthSpeed: 'Su ancho de banda es limitado, estamos reduciendo la calidad de video en función de su ancho de banda disponible.',
+    minCommonChat: 'Ocultar ventana de chat',
+    minoption: 'Ingrese al menos dos opciones',
+    microphoneNotConnected: 'El micrófono no está disponible o no está conectado correctamente',
+    // 'mictesting' : 'Si la barra de audio fluctúa mientras habla, el micrófono está funcionando.',
+    mictesting: 'El micrófono funciona, si la barra de audio anterior fluctúa <br />'
+    + 'y puedes escuchar tu propia voz mientras hablas',
+    minute: 'Minuto',
+    ModeclosingPoll: 'Modo de cerrar la encuesta :',
+    msgForDownloadStart: 'No se pueden guardar los datos. <br /> Preparando datos para descargar',
+    msgForReload: 'Vuelva a cargar esta página para continuar editando.',
+    msgForWhiteboard: 'Pizarra vacía.',
+    msgStudentForReload: 'Por favor, vuelva a cargar esta página.',
+    Multiple_login: 'Error: Acceso denegado, múltiples inicios de sesión',
+    muteAll: 'Silenciar todo',
+    muteAllAudio: 'Silenciar todo el audio',
+    mybsharedoc: 'El documento se compartirá en breve.',
+    NAME: 'NOMBRE',
+    Next: 'Siguiente',
+    note: 'Añadir nota',
+    notePlaceholder: 'Escribe tu nota aquí',
+    page: 'Página',
+    of: 'de',
+    typeQuestion: 'Escriba pregunta',
+    typeOption: 'Opción de tipo',
+    dropVideosHere : 'Suelta videos aquí',
+    nomdlroot: 'No hay URL para Moodle.',
+    normalView: 'Vista normal',
+    noResultStd: 'No tiene permiso para ver el resultado.',
+    NotAllowedError: 'Webcam desactivada',
+    NotFoundError: 'La Webcam no está disponible o no está conectada correctamente.',
+    NotReadableError: 'Asegúrese de que no se esté utilizando la misma webcam <br /> en varios navegadores o varias aplicaciones.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 131. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)